be/src/exec/vectorized/hdfs_scan_node.h

```diff
@@ -169,24 +172,9 @@ class HdfsScanNode final : public starrocks::ScanNode {
     RuntimeProfile::Counter* _io_timer = nullptr;
     RuntimeProfile::Counter* _io_counter = nullptr;
     RuntimeProfile::Counter* _column_read_timer = nullptr;
-    RuntimeProfile::Counter* _level_decode_timer = nullptr;
-    RuntimeProfile::Counter* _value_decode_timer = nullptr;
-    RuntimeProfile::Counter* _page_read_timer = nullptr;
     RuntimeProfile::Counter* _column_convert_timer = nullptr;
 
-    RuntimeProfile::Counter* _bytes_total_read = nullptr;
-    RuntimeProfile::Counter* _bytes_read_local = nullptr;
-    RuntimeProfile::Counter* _bytes_read_short_circuit = nullptr;
-    RuntimeProfile::Counter* _bytes_read_dn_cache = nullptr;
-    RuntimeProfile::Counter* _bytes_read_remote = nullptr;
-
-    // reader init
-    RuntimeProfile::Counter* _footer_read_timer = nullptr;
-    RuntimeProfile::Counter* _column_reader_init_timer = nullptr;
-
-    // dict filter
-    RuntimeProfile::Counter* _group_chunk_read_timer = nullptr;
-    RuntimeProfile::Counter* _group_dict_filter_timer = nullptr;
-    RuntimeProfile::Counter* _group_dict_decode_timer = nullptr;
+    HdfsIOProfile _hdfs_io_profile;
+    HdfsParquetProfile _parquet_profile;
```

Comment:
It seems weird to me that `HdfsScanNode` understand something about file format related code. Should we just extract file format related code to other place.

Reply:
I agree. But it's quite hard now. Profile should be attached to some node, and there is only `HDFSSCanNode`.  And theoretically speaking, a single `HDFSScanNode` could read files of many formats like 'parquet' or 'orc'.

This PR is trying to separate format related profile section from general section. But I don't want to make huge change, just for making profile related section better.

I'd like to like your proposal or suggestion.